packages/orchestration/src/types.ts

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable no-use-before-define -- fine for types */
+// Ambients
+import '@agoric/zoe/exported.js';
+
+import type { Amount, Brand } from '@agoric/ertp/exported.js';
+import type {
+  CoinAmount,
+  CosmosChainInfo,
+  CosmosStakingFacet,
+  Denom,
+  IcaAccount,
+} from './cosmos-api.js';
+import type { EthChainInfo } from './ethereum-api.js';
+import type { ICQConnection } from './exos/icqConnectionKit.js';
+import type {
+  ChainAddress,
+  OrchestrationAccountI,
+  TransferMsg,
+} from './orchestration-api.js';
+
+export type * from './cosmos-api.js';
+export type * from './exos/chainAccountKit.js';
+export type * from './exos/icqConnectionKit.js';
+export type * from './orchestration-api.js';
+export type * from './service.js';
+export type * from './vat-orchestration.js';
+
+/**
+ * static declaration of known chain types will allow type support for
+ * additional chain-specific operations like `liquidStake`
+ */
+export type KnownChains = {
+  stride: {
+    info: CosmosChainInfo;
+    methods: {
+      liquidStake: (amount: AmountArg) => Promise<void>;
+      // ??? do we prefer to merge capabilities into a single object? If we do
+      // that, then the chain-native methods (e.g. delegate) are hard to
+      // distinguish from generic. Alternately, there could be a method to get a
+      // generic facade on the chain-native methods.
+      getIcaAcccount: () => IcaAccount;
+      getIcqConnection: () => ICQConnection;
+      getStakingFacet: () => CosmosStakingFacet;
+    };
+  };
+  cosmos: { info: CosmosChainInfo; methods: {} };
```

Comment:
shoudl have most of the methods of celestia?

Reply:
I wrote an XXX instead of tackling this now. I expect these will be autogenerated